### PR TITLE
SPM Support

### PR DESCRIPTION
**Pull request by mustafagunes**
Hello friends👋🏻 , when I saw the library rising fast, I wanted to contribute. I added SPM support. I hope it made your job easier.

### CODE REVIEW

**Comment by mustafagunes** [comment]
cc: @demirciy @Bugrayenidunya @ferhanakkan

File: README.md [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
 [![CI Status](https://github.com/Retro-Cartoons/Tom/workflows/Tom%20CI/badge.svg?branch=main)](https://github.com/Retro-Cartoons/Tom/actions)
 [![Version](https://img.shields.io/cocoapods/v/Tom.svg)](https://github.com/Retro-Cartoons/Tom)
+[![SPM Compatible](https://img.shields.io/badge/SPM-1.0.0-brightgreen.svg)](https://swift.org/package-manager)
 [![License](https://img.shields.io/cocoapods/l/Tom.svg)](https://github.com/Retro-Cartoons/Tom)
 [![Platform](https://img.shields.io/cocoapods/p/Tom.svg)](https://github.com/Retro-Cartoons/Tom)
 
 - [Demo](#demo)
 - [Requirements](#requirements)
 - [Installation](#installation)
+  - [SPM](#Swift Package Manager)
```
Comment (meyusufdemirci) [inline README.md]:
```suggestion
  - [SPM](#swift-package-manager)
```

That will help the jumping to enable

File: README.md [hunk 2]
```diff
@@ -26,6 +28,11 @@
 
 ## Installation
 
+### Swift Package Manager
+
+**Swift Package Manager (requires Xcode 11)**
+Add package into Project settings -> Swift Packages -> Repo URL
+
```
Comment (meyusufdemirci) [inline README.md:37]: We already support min Xcode 11, therefore no need to mention `(requires Xcode 11)`.

What about,

```suggestion
1. File > Swift Packages > Add Package Dependency
2. Add `https://github.com/Retro-Cartoons/Tom.git`

_OR_

Update `dependencies` in `Package.swift`
```swift
dependencies: [
    .package(url: "https://github.com/Retro-Cartoons/Tom.git", .upToNextMajor(from: "1.0.0"))
]
```
```

**Review by meyusufdemirci — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mustafagunes) [reply README.md:37]: You're absolutely right. It's more understandable that way.

File: Package.swift [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// swift-tools-version:5.1
+
+import PackageDescription
+
+let package = Package(
+    name: "Tom",
+    platforms: [
+        .iOS(.v13)
```
Comment (mustafagunes) [inline Package.swift:8]:
```suggestion
        .iOS(.v11)
```
Here I added the platform version number incorrectly. Sorry 🙃
cc: @mustafagunes

File: Package.swift [hunk 4]
```diff
@@ -0,0 +1,17 @@
+// swift-tools-version:5.1
+
+import PackageDescription
+
+let package = Package(
+    name: "Tom",
+    platforms: [
+        .iOS(.v11)
```
Comment (meyusufdemirci) [inline Package.swift]: We support min 13. Please revert it to 13.

**Review by meyusufdemirci — CHANGES_REQUESTED** [review]

**Comment by meyusufdemirci** [comment]
Thanks for your effort but we can't merge it since you created the branch from `main` which is not the latest branch. It produces a lot of conflicts then. I will create a new PR with your changes.

**Comment by meyusufdemirci** [comment]
New PR https://github.com/Retro-Cartoons/Tom/pull/27

**Pull request closed without merging**